website/docs/tutorials/structured_config/5_schema.md

```diff
@@ -2,11 +2,11 @@
 id: schema
 title: Structured config schema
 ---
-We have seen how to use Structured Configs as configuration, but they can also be used as a schema validating configuration files!  
-When Hydra loads a configuration, it looks for config with the same name in the `ConfigStore`.
+We have seen how to use Structured Configs as configuration, but they can also be used as schema validating configuration files!
+When Hydra loads a configuration file, it looks for a config with the same name in the `ConfigStore`.
```

Comment:
hmm, not accurate.
the configuration can be provided by other config sources that are not file based.

Reply:
Ok will revert the "file" part, however I'm curious how else you can provide a config? The two ways I'm aware of are (1) through a file, or (2) through the config store, but here (2) wouldn't make sense, would it?

Reply:
The config store would not make sense, true.
You can provide it as a file (actual file in the file system) and as a resource (file packaged with your par).
there is an abstraction for ConfigSource that is currently not supporting anything else but I did have a prototype that fetched config files directly from GitHub using some API (it was just too slow to be useful).

You could also imagine configs in some config storage system (configerator), or configs based on a database or some other data storage like LDAP.
the sky is the limit :)

Reply:
Ok thanks => I removed the "file" in 16ef8dad